scripts/utils.js

```diff
@@ -0,0 +1,53 @@
+const hre = require('hardhat');
+
+const networks = {
+  mainnets: [
+    'mainnet',
+    'arbitrum',
+    'avalanche',
+    'bsc',
+    'fantom',
+    'gnosis',
+    'metis',
+    'milkomeda',
+    'moonbeam',
+    'moonriver',
+    'optimism',
+    'polygon',
+    'rsk',
+  ],
+  testnets: [
+    'goerli',
+    'sepolia',
+    'avalanche_testnet',
+    'milkomeda_testnet',
+    'fantom_testnet',
+    'moonbase_testnet',
+    'bsc_testnet',
+    'rsk_testnet',
+    'polygon_testnet',
+    'arbitrum_testnet',
+    'optimism_testnet',
+    'gnosis_testnet',
+  ],
+};
+
+const apis = {
```

Comment:
`apis` would have been an appropriate name for `['ANU Quantum Random Numbers', 'byog']`, bad naming. Also, I wouldn't define this separately and just embed this data in an `if` statement in the function below.